### PR TITLE
changed header/footer option for Chrome 120

### PR DESCRIPTION
**Pull request by alpacathunder1**
Fixes https://github.com/mikepqr/resume.md/issues/41

### CODE REVIEW

**Comment by mikepqr** [comment]
Thanks!

Would be nice to keep backwards compatibility with older Chrome versions.

Can you keep the `--print-to-pdf-no-header` option too or does passing that option and the new one simultaneously break new versions?

If you can keep it then please do, and add a comment saying something like `Keep both versions of this option for backwards compatibility, https://developer.chrome.com/docs/chromium/new-headless`

**Comment by mikepqr** [comment]
Sorry, I confused you by posting on the original issue thread rather than this PR!

Can you please add that comment to resume.py (not to this PR)?

File: resume.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ def write_pdf(html: str, prefix: str = "resume", chrome: str = "") -> None:
         "--no-sandbox",
         "--headless",
         "--print-to-pdf-no-header",
```
Comment (mlw0) [inline resume.py:130]: Add the comment here, to this file, in the diff :-)

Comment (mikepqr) [inline resume.py:130]: Add the comment here, to this file, in the diff :-)

File: resume.py [hunk 2]
```diff
@@ -128,6 +128,7 @@ def write_pdf(html: str, prefix: str = "resume", chrome: str = "") -> None:
...
+        "--no-pdf-header-footer",
```
[the remark this answers is not in the document]
Comment (mikepqr) [reply resume.py:131]: I'm asking for a Python comment, i.e. a line starting with `#`, in the diff itself. Not a comment on github.com.

[the remark this answers is not in the document]
Comment (mikepqr) [reply resume.py:133]: All set. Thank you for the contribution!

**Pull request merged**